.vscode/settings.json

```diff
@@ -128,5 +128,8 @@
     "whrf",
     "whrt",
     "wintel"
-  ]
+  ],
+  "rust-analyzer.cargo.features": [
+    "bert"
```

Comment:
won't this disturb e.g. working on tremor-script alone?

Reply:
Sorry about that, that slipped through I thought vscode was in ignore!

Reply:
fixed